### PR TITLE
fix(cli): find axe-core when installed globally

### PR DESCRIPTION
**Pull request by michael-siek**
See: #229 
This issue only happens when installing `@axe-core/cli` globally.

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [ ] Code is reviewed for security

### CODE REVIEW

**Review by stephenmathieson — CHANGES_REQUESTED** [review]
I think we could better solve this using Node's module lookup algorithm.

```
axe_core_file_path = require.resolve('axe-core')
```

This will return the _nearest_ installed axe-core. It works both when installed globally and as a project dependency.

**Review by WilcoFiers — DISMISSED** [review]
Tests

**Comment by stephenmathieson** [comment]
Writing a proper test for this will be difficult. To actually reproduce the problem, we'd have to globally install the program and run it. Not really something that qualities as a "unit test" IMO. Furthermore, we're using Node's algorithm now, which we already assume works (otherwise, nothing here would). Since the existing tests pass, and this is _removing_ custom logic and replacing it with Node's `require`, do we really need to setup an E2E test for this?

File: packages/cli/src/lib/utils.test.ts [hunk 1]
```diff
@@ -71,6 +72,24 @@ describe('utils', () => {
...
+          }
+        });
+      });
+
+      afterEach(() => {
+        mock.restore();
+      });
+      it('fall back to use `locally` installed axe-core', () => {
+        const axeSource = utils.getAxeSource();
+        assert.isNotNull(axeSource);
```
Comment (WilcoFiers) [inline packages/cli/src/lib/utils.test.ts:90]: I'm not sure what this is supposed to prove, but just asserting that it isn't null doesn't mean much.

**Review by WilcoFiers — CHANGES_REQUESTED** [review]

**Review by WilcoFiers — APPROVED** [review]

**Pull request merged**